docs/src/main/sphinx/connector/pulsar.rst

```diff
@@ -0,0 +1,270 @@
+=================
+Pulsar connector
+=================
+`Apache Pulsar <https://pulsar.apache.org/en/>`_ is a cloud-native, distributed messaging and streaming platform.
+
+This connector allows the use of Pulsar topics as tables in Trino, such that each Pulsar message
+in a Pulsar topic is presented as a row in Trino. And Pulsar topic schema is mapped to Trino
+table schema.
+
+Topics in Pulsar are stored as segments in `Apache BookKeeper <https://bookkeeper.apache.org/>`_. Each topic segment is replicated to some BookKeeper nodes,
+which enables concurrent reads and high read throughput. You can configure the number of BookKeeper nodes,
+and the default number is 3. In Trino Pulsar connector, data is read directly from BookKeeper,
+so Trino workers can read concurrently from horizontally scalable number BookKeeper nodes.
+
```

Comment:
Probably here insert a Requirements section modelled in wording after e.g. the sqlserver.rst currently in master

Reply:
Yes, we've added the Requirements section in the draft and need your review later, thanks